test/unit/crypto/keys.test.ts

```diff
@@ -0,0 +1,42 @@
+import chai, { expect } from "chai";
+import sinonChai from "sinon-chai";
+import {Wallet} from "../../../src/interfaces";
+import sinon from "sinon";
+import {generateKeys} from "../../../src/crypto/keys";
+import {InvalidSeedError} from "../../../src/crypto/error";
+
+chai.use(sinonChai);
+
+describe('generateKeys', () => {
+
+  const sandbox = sinon.createSandbox();
+  const walletStub = {} as Wallet;
```

Comment:
in separate file implement MockWallet implement Wallet and implement every method like this:
`public updatePluginState = sinon.stub();`